open_prices/api/proofs/views.py

```diff
@@ -99,8 +118,13 @@ def upload(self, request: Request) -> Response:
         serializer.is_valid(raise_exception=True)
         # get source
         source = get_source_from_request(self.request)
+        # Here, user is either a `open_prices.users.models.User` (if
+        # authenticated) or an `django.contrib.auth.models.AnonymousUser`
+        # (if not authenticated)
+        user = self.request.user
+        owner = None if user.is_anonymous else user.user_id
```

Comment:
None ? Shouldn't we set it to `openfoodfacts-contributors` instead ?

Reply:
After a bit of thought, it was easier to add a new flag (`anonymous`) instead of using a hardcoded owner.

Reply:
I'd rather mimick what is done in OFF.
Also avoids an extra field (with extra rules where is_anonymous can only be True if no owner is set, etc).